wonderful_bing/wonderful_bing.py

```diff
@@ -37,8 +39,11 @@
 class WonderfulBing(object):
     def __init__(self, arguments):
         # Get all the information we need from this url, see issue#7
+        self.country = arguments['--country']
+        if not self.country:
+            self.country = 'en-US'
```

Comment:
It seems it won't raise `keyerror` this way, I tried. And look at line 120: arguments['--directory']. The `or` way is great and I will update PR.